go/src/koding/kites/kloud/provider/azure/bootstrap_test.go

```diff
@@ -0,0 +1,28 @@
+package azure_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"koding/kites/kloud/provider/azure"
+)
+
+func TestBootstrapJSONTmpl(t *testing.T) {
+	cases := []*azure.BootstrapConfig{
+		{Rule: false},
+		{Rule: true},
+	}
+
+	for i, cas := range cases {
+		tmpl, err := azure.NewBootstrapTmpl(cas)
+		if err != nil {
+			t.Fatalf("%d: NewBootstrapTmpl()=%s", i, err)
+		}
+
+		var v map[string]interface{}
+
+		if err = json.Unmarshal(tmpl, &v); err != nil {
```

Comment:
I don't test for expected values, only for valid JSON - so for every case throwaway `v` variable is used.